spotbox/playback.py

```diff
@@ -1,56 +1,67 @@
-#!/usr/bin/python
-
 """SPOTBOX media playback backend"""
 
-# Note-- non-iTunes mode is not yet fully implemented.
+import os
+import stat
+import subprocess
+import time
+
 
 
 class Playback:
     """A playback object, allows for audio files to be loaded, played back,
     and stopped whilst playing.
     """
-    def __init__(self):
-        pass
+    def load(self, spotnumber, filepath):
+        raise NotImplementedError
 
-    def initialize_one_player(self, spotnumber):
-        pass
+    def play(self, spotnumber):
+        raise NotImplementedError
 
-    def load(self, spotnumber, filepath):
-        pass
+    def stop(self):
+        raise NotImplementedError
+
+
+class StubPlayback(Playback):
+
+    def load(self, spotnumber, spot):
+        print('LOADING {} ({})'.format(spotnumber, spot))
 
     def play(self, spotnumber):
-        pass
+        print('PLAYING {}'.format(spotnumber))
 
     def stop(self):
-        pass
+        print('STOPPING')
 
 
 class iTunesPlayback(Playback):
```

Comment:
Just ditch it? If modern iTunes have stopped working, I don't see why not end-of-life it.